azurerm/resource_arm_cosmos_db_account_test.go

```diff
@@ -110,6 +110,27 @@ func TestAccAzureRMCosmosDBAccount_eventualConsistency(t *testing.T) {
 	})
 }
 
+func TestAccAzureRMCosmosDBAccount_mongoDB(t *testing.T) {
+	resourceName := "azurerm_cosmosdb_account.test"
+	ri := acctest.RandInt()
+	config := testAccAzureRMCosmosDBAccount_mongoDB(ri, testLocation())
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testCheckAzureRMCosmosDBAccountDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: config,
+				Check: resource.ComposeTestCheckFunc(
+					testCheckAzureRMCosmosDBAccountExists(resourceName),
+					resource.TestCheckResourceAttr(resourceName, "kind", "MongoDB"),
```

Comment:
It would be nice to add a similar check to existing tests - just to verify that we expose the default kind.

Reply:
Done